pages/agent/v3/windows.md.erb

```diff
@@ -50,6 +44,23 @@ See the [Agent SSH Keys](/docs/agent/v3/ssh-keys) documentation for more details
 
 The configuration file is located at `C:\buildkite-agent\buildkite-agent.cfg`. See the [configuration documentation](/docs/agent/v3/configuration) for an explanation of each configuration setting.
 
+There are two options to be aware of for this initial setup:
+
+* Set your [agent token](/docs/agent/v3/tokens), if you did not set it as an environment variable during installation.
+* You may need to use the `shell` configuration option. On Windows, Buildkite defaults to using Batch. If you want to use PowerShell or Bash, you must point Buildkite to the correct shell. For example, to use PowerShell:
+
+    ```cfg
+    shell="<path-to-powershell-executable>"
+    ```
+
+    <div class="Docs__note">
+    <p>This allows you to run a PowerShell script, but your build hooks must still be either `.bat` or `.sh` files (`.sh` files require access to Git for Windows). Refer to <a href="https://buildkite.com/docs/agent/v3/hooks#hooks-on-windows">Hooks - Hooks on Windows</a> for more information.</p>
```

Comment:
In great sadness, thse markdown `code` need to be HTML cos they are inside the note.

That is a thing I want to fix.